gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
+      fileName: './dist/mdl.closure.min.js',
```

Comment:
Sorry, I cleared all of `node_modules`, reinstalled and it still showed up in the root dir.

If you're happy to merge we can fix it later.